public/app/percona/dbaas/components/DBCluster/PSMDB.service.ts

```diff
@@ -133,7 +143,7 @@ export class PSMDBService extends DBClusterService {
       cpu: (dbCluster.params.replicaset?.compute_resources?.cpu_m || 0) / THOUSAND,
       disk: (dbCluster.params.replicaset?.disk_size || 0) / BILLION,
       status: dbCluster.state || DBClusterStatus.changing,
-      message: dbCluster.operation?.message,
+      message: dbCluster.operation?.message, // TODO 11031 посмотреть, есть ли в новом апи
```

Comment:
Let's go with English here 😂

Reply:
I'm sorry! There were so many files and changes in the form already in progress, I lost focus 🙈
I fixed